packages/cozy-pouch-link/src/jsonapi.js

```diff
@@ -1,6 +1,7 @@
 export const normalizeDoc = (doc, doctype) => {
   const id = doc._id || doc.id
-  return { id, _id: id, _type: doctype, ...doc }
+  const rev = doc._rev || doc.rev
+  return { id, _id: id, rev, _rev: rev, _type: doctype, ...doc }
```

Comment:
I suggeste to not create the `rev` key but use always `_rev`:
```suggestion
  const normalized = { ...doc, id, _id: id, _rev: rev, _type: doctype }
  delete normalized.rev

  return normalized
```

Reply:
Done !